astropy/time/core.py

```diff
@@ -1441,13 +1441,28 @@ def argmax(self, axis=None, out=None):
 
         return dt.argmax(axis, out)
 
-    def argsort(self, axis=-1):
+    def argsort(self, axis=-1, kind="stable"):
         """Returns the indices that would sort the time array.
 
-        This is similar to :meth:`~numpy.ndarray.argsort`, but adapted to ensure
-        that the full precision given by the two doubles ``jd1`` and ``jd2``
-        is used, and that corresponding attributes are copied.  Internally,
-        it uses :func:`~numpy.lexsort`, and hence no sort method can be chosen.
+        This is similar to :meth:`~numpy.ndarray.argsort`, but adapted to ensure that
+        the full precision given by the two doubles ``jd1`` and ``jd2`` is used, and
+        that corresponding attributes are copied.  Internally, it uses
+        :func:`~numpy.lexsort`, and hence no sort method can be chosen.
+
+        Parameters
+        ----------
+        axis : int, optional
+            Axis along which to sort. Default is -1, which means sort along the last
+            axis.
+        kind : 'stable', optional
```

Comment:
Is "stable" the only acceptable option now? If not, please list other options.

Reply:
OK, I can do `**kwargs`, makes sense. But maybe let @mhvk weigh in before doing that.

Reply:
At first, I wondered whether we should just let the defaults for `Column.argsort` and `Table.argsort` become stable, but looking further, for `Time` we made a bit of an effort to make sure the various `ndarray`-like methods have signatures similar to those of `ndarray`. E.g., we include `out=None` even though we don't allow passing in an output `Time`. So, I think on balance I'd suggest just sticking with what you have here.